src/components/WebexMember/WebexMember.scss

```diff
@@ -12,9 +18,17 @@
     padding-left: 0.25rem;
   }
 
-  .member-muted {
-    flex: 1;
+  .muted {
     text-align: right;
     color: #ff0000;
   }
+
+  .sharing {
+    padding-right: 2.5rem;
+    line-height: 1.5rem;  
+  }
+
+  .presenter {
+    padding-left: 0.25rem;
```

Comment:
let's move the padding to the `.descripton` div, would this make sense?

Reply:
yes, that makes sense. I moved the property to the description class and also turned it into the left margin. ;)